extension/src/Documentation.ts

```diff
@@ -1234,6 +1186,255 @@ export async function generateExternalDocumentation(
+          control.name
+        } | ${readOnly ? "yes" : ""} |${
+          printSummary
+            ? ` ${
+                control.xmlComment?.summary
+                  ? ALXmlComment.formatMarkDown({
+                      text: control.xmlComment.summary,
+                      inTableCell: true,
+                    })
+                  : ""
+              } |`
+            : ""
+        }\n`;
+      });
+      if (controlsContent !== "") {
+        objectIndexContent += `## Controls\n\n`;
+        objectIndexContent += `| Type | Name | Read-only |${
+          printSummary ? " Description |" : ""
+        }\n`;
+        objectIndexContent += `| ---- | ------- | ----------- |${
+          printSummary ? " ------------- |" : ""
+        }\n`;
+        objectIndexContent += controlsContent;
+        objectIndexContent += "\n";
+      }
+      return objectIndexContent;
+    }
+
+    function getWsPageFieldsTable(object: ALObject): string {
+      let objectIndexContent = "";
+      const allControls = object.getAllControls();
+      let controls = allControls.filter(
+        (control) =>
+          control.type === ALControlType.pageField ||
+          control.type === ALControlType.part
+      );
+
+      controls = controls.filter((c) => !c.isObsoletePending(false));
+
+      let controlsContent = "";
+      const printSummary =
+        controls.find((x) => x.xmlComment?.summary !== undefined) !== undefined;
+      controls.forEach((control) => {
+        const readOnly =
+          control.type === ALControlType.part
+            ? (control as ALPagePart).readOnly
+            : (control as ALPageField).readOnly;
+
+        controlsContent += `| ${controlTypeToText(control)} | ${
+          control.name
+        } | ${readOnly ? "yes" : ""} |${
+          printSummary
+            ? ` ${
+                control.xmlComment?.summary
+                  ? ALXmlComment.formatMarkDown({
+                      text: control.xmlComment.summary,
+                      inTableCell: true,
+                    })
+                  : ""
+              } |`
+            : ""
+        }\n`;
+      });
+      if (controlsContent !== "") {
+        objectIndexContent += `## Controls\n\n`;
+        objectIndexContent += `| Type | Name | Read-only |${
+          printSummary ? " Description |" : ""
+        }\n`;
+        objectIndexContent += `| ---- | ------- | ------- |${
+          printSummary ? " ------------- |" : ""
+        }\n`;
+        objectIndexContent += controlsContent;
+        objectIndexContent += "\n";
+      }
+      return objectIndexContent;
+    }
+
+    function getEnumValuesTable(object: ALObject): string {
+      let objectIndexContent = "";
+      const values = (object.controls.filter(
+        (o) => o.type === ALControlType.enumValue
+      ) as ALEnumValue[]).filter(
+        (o) => !o.isObsoletePending() && !o.isObsolete()
+      );
+      if (values.length > 0) {
+        objectIndexContent += `## Values\n\n`;
```

Comment:
How come back tics are used on this line? Same with 1278, 1326, 1375.

Reply:
No reason at all (anymore)... Strange that Prettify doesn't complains...

Reply:
I found like 30 more places in this file only... 😁